nixos/modules/services/networking/murmur.nix

```diff
@@ -109,6 +109,12 @@ in
         description = "Host to bind to. Defaults binding on all addresses.";
       };
 
+      package = mkOption {
+        type = types.package;
+        default = pkgs.murmur;
```

Comment:
Set `defaultText = "pkgs.murmur"`. Otherwise it renders like this:

![2021-01-10_23-24](https://user-images.githubusercontent.com/20525370/104137136-055dc600-539b-11eb-8c36-d48cee8575cf.png)

Reply:
Thanks! Fixed.